### DISCUSSION TITLE
Welcome to mrt-buddy Discussions!

### DISCUSSION THREAD

**Discussion by aniruddha-adhikary** [aniruddha-adhikary/mrt-buddy#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sadman-123** [comment]
Hello!
This app is great..may i give you some suggestions:
It will be Great if you can show a Map or like Stepper to view the metro stations where i have travelled

**Comment by abdulazizzisan** [comment]
This message is mainly towards aniruddha-adhikary, hmasum52 and rahat14 brother. 
I want to contribute to this project more and want to know more about this project. Specially about the NFC technology. How the NFC was detected, how it's read, how the data is parsed and converted into text/string. 
I've been learning Android Development for quite some time now but everything used in this project is new for me. So I am really interested to learn about the technology behind it so that I can work on it in the future.
It would be great if you could share some resources with me. I'll be grateful to you for the rest of my life brothers.
I've tried to connect with you personally but couldn't achieve that and that's why I'm writing this here.

**Comment by alaminsoyaib** [comment]
Can it read "Rapid Pass"  cards too?
